security_check/main.py

```diff
@@ -159,8 +159,19 @@ def _check_vuln_is_valid(vuln):
     return False
 
 
-def _main():
+def _security_check(image, severity=_MEDIUM, whitelist_file='whitelist.json'):
```

Comment:
One last nit, you can drop the leading underscore since we're exposing this more broadly now.